### DISCUSSION TITLE
Document placeholders like $Key and $File in MoveToResource template and CodeReference regex

### DISCUSSION THREAD

**Discussion by eng-marani** [dotnet/ResXResourceManager#553] [Q&A]
I see from examples that MoveToResource and CodeReference Regexp uses some "placeholderd" that are replaced by real content data.
I have found no trace of them into documentation. Just i can reverse engineer that $Key is the resource key and $File is the source file.

I'm trying to create a custom "MoveToReference" that generate a custom language function, where you can pass a kay but also the original text as fallback when the key is not present. But i do not know if some other placeholders like $Text or $Value are available.

Is it possible to document somewhere into MoveToResource and CountCodeReference the usage of these placeholders ?

How can i create a custom function that replace also the original text so it can be used as a default parameter using MoveToResource tool ?

**Comment by tom-englert** [comment]
No, there are no other variables available.

PRs with documentation improvements are always welcome.
